EIPS/EIP-873-draft_steroid_network.md

```diff
@@ -0,0 +1,44 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: EIP-873: Steroid Network
+    Author: Angelos Hadjiphilippou <angelos@exevior.com>
+    Type: Standard Track
+    Category (*only required for Standard Track): Core 
+    Status: Draft
+    Created: 2018-02-06
+    Requires (*optional): <EIP number(s)>
+    Replaces (*optional): <EIP number(s)>
+
+## Simple Summary
+The suggestion implies a build-in pool and a change in the way new blocks are handled in the following manner. New block found -> Network broadcasts new pending block to miners, miners process, change status from pending to complete & add to chain.
+
+## Abstract
+
+Scalability to the number of transactions processed is a threshold subject to a number of factors, some of which are current network, diff etc, not to mention the 51% pool dominance issue. This EIP takes away the 51% issue and makes number of transactions per second only liable to hashrate availability and also considers a fairer distribution of rewards only based on work submitted and not just "luck"!
+
+## Specification
+
+* Steroid network pool build-in to core
+*Implies that all miners are connected directly and not to 3rd-party pools
+
+The Steroid pool gathers all the newly found blocks and applies a "Pending" status to them. It then broadcasts to the network for miners to pick-up and process. The Steroid pool can control the number of pending blocks required, for the current transaction load, dynamically using the traditional diff method.
```

Comment:
By the way, which chain ID will the Steroid network use?

Reply:
I'm researching to see if we can Incorporate the existing framework and logic without any fundamental core changes, so bare with me on that one!

Reply:
Until you choose an approach, the draft seems not implementable.